vyper/parser/expr.py

```diff
@@ -674,26 +674,53 @@ def call(self):
 
         if isinstance(self.expr.func, ast.Name):
             function_name = self.expr.func.id
+
             if function_name in dispatch_table:
                 return dispatch_table[function_name](self.expr, self.context)
+
+            # Struct constructors do not need `self` prefix.
+            elif function_name in self.context.structs:
+                # Can also be called in return statement...
+                #if not self.context.in_assignment and not self.context.:
+                #    raise StructureException("Struct constructor must be called in RHS of assignment.", self.expr)
```

Comment:
why was this exception commented out?

Reply:
Struct constructors can also be called in return statements